e2e/trust/sign_test.go

```diff
@@ -0,0 +1,90 @@
+package trust
+
+import (
+	"os"
+	"fmt"
+	"testing"
+
+	"github.com/gotestyourself/gotestyourself/fs"
+	"github.com/gotestyourself/gotestyourself/icmd"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	notaryURL    = "https://notary-server:4443"
+	alpineImage  = "registry:5000/alpine:3.6"
+	alpineSha    = "641b95ddb2ea9dc2af1a0113b6b348ebc20872ba615204fbe12148e98fd6f23d"
+	busyboxImage = "registry:5000/busybox:1.27.2"
+	busyboxSha   = "030fcb92e1487b18c974784dcc110a93147c9fc402188370fbfd17efabffc6af"
+	localImage   = "registry:5000/signlocal:v1"
+	signImage    = "registry:5000/sign:v1"
+)
+
+func TestSignWithTrustSign(t *testing.T) {
```

Comment:
What is "WithTrustSign" ? What is the case that is being tested here?  Is it just a local image? `TestSignLocalImage` ?

Reply:
as opposed to DCT=1 push, I can change it.